### ISSUE TITLE
Live Tv external app bug

### ISSUE THREAD

**Issue by MichaelRUSF** [jellyfin/jellyfin-androidtv#2326]
### Describe the bug

If I set my preferred media player to "External app" for Live Tv, then select a Live Tv channel, the program will just go to a black screen. After a few seconds the message "Cannot play this video format" pops up briefly, then goes away leaving only the black screen. The only option after this is to back out into the main menu, the system does not crash.

In my server's transcode folder in the Jellyfin program data directory, I will find that there is an ever-growing .ts file streaming the live tv channel. This will continue to increase in size until there is no more hard drive space left. Even if I restart the Jellyfin AndoidTv app or Tv the server continues to stream to the .ts file. Only restarting the server stops the stream.

I have tested this on the last stable build (.14.5) and this issue does not occur. If I choose my preferred media as an external app for live tv, it will prompt me for an external app when I select a Live Tv channel as expected. Currently using build #2900 produces this. Posted relevant server logs.

### Application version

(0.15.0-beta3+) build #2900

### Where did you install the app from?

Sideloaded APK

### Device information

Sony Bravia 4k VH2

### Android version

Android TV OS 10

### Jellyfin server version

10.8.8

**Comment by nielsvanvelzen** [comment]
Please provide client logs

**Comment by MichaelRUSF** [comment]
I'm only able to provide logs from the server as this bug does not lead to a crash, so there are no logs that I can find. If there is a way I can access some debugging logs from the androidtv client, please let me know and I can try to access them.
 
[log_20221208.log](https://github.com/jellyfin/jellyfin-androidtv/files/10188689/log_20221208.log)

**Comment by MichaelRUSF** [comment]
I was able to obtain the log below from my Tv. 
[logcat.txt](https://github.com/jellyfin/jellyfin-androidtv/files/10190695/logcat.txt)

**Comment by nielsvanvelzen** [comment]
Those log don't show you opening an external player so I can't see much in them. It does look like your websocket connection fails but that's unrelated to this issue.

**Comment by MichaelRUSF** [comment]
Yep, that's the issue. It never gives me an option for an external player it just goes straight to a black screen and then spits out the error message then back to a black screen. The websocket error was b/c I shut down the server before collecting the logs so I could delete the .ts file, which is not related to the issue. I wanted to delete that .ts file right away. When I first discovered the bug, my primary drive only had 5 gigs free b/c the server had been streaming to the file for a few hours.

**Comment by nielsvanvelzen** [comment]
Ok let me rephrase that, the logs don't show any attempt to opening a video. It only shows the app going back to the home screen. So that's why I can't see anything useful in the logs.

**Comment by MichaelRUSF** [comment]
I was able to get a better log file by tuning to the channel from the live tv guide as opposed to the home screen. Same results, except this time the error message states "Cannot play this video format". 

[logcat livetv external.txt](https://github.com/jellyfin/jellyfin-androidtv/files/10199267/logcat.livetv.external.txt)

**Comment by nielsvanvelzen** [comment]
The app is unable to determine a stream URL so it can't play anything. That's why the external player doesn't work. If someone wants to work on this issue here's what I found so far.

An exception is thrown in this if statement:

https://github.com/jellyfin/jellyfin-androidtv/blob/d3b1faeb993f3e35b7befee48b542a4ecf02fa07/app/src/main/java/org/jellyfin/androidtv/ui/playback/GetPlaybackInfoResponse.java#L180-L185

This is the exception with stacktrace from the provided logs:

```
2022-12-10 00:14:25.268 19171-19171/org.jellyfin.androidtv.debug I/jellyfin-apiclient: Response received from: http://192.168.1.39:8096/LiveStreams/Open?format=json
2022-12-10 00:14:25.270 19171-19171/org.jellyfin.androidtv.debug E/ExternalPlayerActivity: Error getting playback stream info
    org.jellyfin.androidtv.data.compat.PlaybackException
        at org.jellyfin.androidtv.ui.playback.GetPlaybackInfoResponse.onResponseInternal(GetPlaybackInfoResponse.java:181)
        at org.jellyfin.androidtv.ui.playback.GetPlaybackInfoResponse.access$000(GetPlaybackInfoResponse.java:23)
        at org.jellyfin.androidtv.ui.playback.GetPlaybackInfoResponse$1.onResponse(GetPlaybackInfoResponse.java:71)
        at org.jellyfin.androidtv.ui.playback.GetPlaybackInfoResponse$1.onResponse(GetPlaybackInfoResponse.java:65)
        at org.jellyfin.apiclient.interaction.SerializedResponse.onSerializedResponse(SerializedResponse.java:46)
        at org.jellyfin.apiclient.interaction.SerializedResponse.onResponse(SerializedResponse.java:39)
        at org.jellyfin.apiclient.interaction.SerializedResponse.onResponse(SerializedResponse.java:6)
        at org.jellyfin.apiclient.interaction.ApiClientRequestListener.onResponse(ApiClientRequestListener.java:21)
        at org.jellyfin.apiclient.interaction.ApiClientRequestListener.onResponse(ApiClientRequestListener.java:5)
        at org.jellyfin.apiclient.interaction.VolleyResponseListener.onResponse(VolleyResponseListener.java:22)
        at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:82)
        at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:29)
        at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:102)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:932)
```

The onResponseInternal method didn't see much changes in the 0.15 development cycle (only a few migrations to the SDK but those are pretty much 1 to 1 conversions).
